### ISSUE TITLE
Link missing a text alternative (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#137]
6 occurrences in one page ([https://newton.dlib.indiana.edu/page/chem-lab](https://newton.dlib.indiana.edu/page/chem-lab)). There's an empty `<a href>` tag the precedes the section headings for each video. Deleting those should take care of the problem. If the empty links do something like facilitate anchoring, then the link will need an "alt" attribute with an explanation.

Related to [https://github.com/IUBLibTech/newton_chymistry/issues/136](https://github.com/IUBLibTech/newton_chymistry/issues/136) (same HTML page with issues)

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [x] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
These six `<a href="">` tags do not actually function anything. For example, `<a href="#gold">` is for jumping to `<xxx name="gold">` in the same page, but there are no destinations corresponding to `<a href="#gold">`. Therefore, I will simply delete them.

<img width="550" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/043bc00b-d09c-4f43-86a8-e7b6d812f814">
